common/types/src/monitoring.rs

```diff
@@ -0,0 +1,122 @@
+use std::{collections::HashSet, sync::LazyLock, time::SystemTime};
+
+use nym_crypto::asymmetric::identity::{PrivateKey, PublicKey, Signature};
+use nym_mixnet_contract_common::MixId;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+static NETWORK_MONITORS: LazyLock<HashSet<String>> = LazyLock::new(|| {
+    let mut nm = HashSet::new();
+    nm.insert("5VsPyLbsBCq9PAMWmjKkToteVAKNabNqex6QwDf5fWzt".to_string());
```

Comment:
what's with this hardcoded value here?

Reply:
Thats the current public key of the network monitor that is allowed to submit stuff